packages/core/src/App/Constants/routes-config.js

```diff
@@ -61,7 +61,7 @@ const getModules = ({ is_deriv_crypto }) => {
             getTitle: () => localize('Bot'),
         },
         {
-            path: routes.dashboard,
+            path: is_dashboard ? routes.dashboard : routes.dashboard_home,
```

Comment:
is this for accessing the dashboard when `is_dashboard` is false? :thinking:

Reply:
@reza-fs just sorted this.